plugins/module_utils/dnac.py

```diff
@@ -82,9 +82,12 @@ def __init__(self, module):
         # To add a new version, simply update the 'dnac_versions' dictionary with the new version string as the key
         # and the corresponding version number as the value.
         self.dnac_versions = {
```

Comment:
I suggest to use in APIs.

Keep 
`self.dnac_version = self.payload.get("dnac_version")
`
And then, set the integer version
```
set_dnac_version_int(self)
    self.dnac_version_integer = int(self.dnac_version.replace(".", ""))
```


We also need to change whereever self.dnac_version is being used.. and we need to use only get method to get the values.